doc/wiki.txt

```diff
@@ -1960,8 +1960,8 @@ MAPPINGS AND COMMANDS REFERENCE                       *wiki-mappings-reference*
     in-browser wiki navigation, but for it to work well, it requires that:
 
       1. `-ext` is set to `html`, and
-      2. |g:wiki_link_creation| should have a `url_transform` that appends
-         the extension.
+      2. |g:wiki_link_creation| should have an entry with the key `html`,
```

Comment:
Hmm. No, I don't think it's right that it needs an entry with the key `html`, thus I think this PR is wrong. However, we need that the entry that is selected from `g:wiki_link_creation` has the `url_transform` key.